docker-compose.yml

```diff
@@ -4,6 +4,7 @@ services:
     image: owlplace
     ports:
       - "3001:3000"
+      - "3009:3000"
```

Comment:
why are we remapping another host port to 3000?

Reply:
we were confused but now we are taking these lines out